### PR TITLE
fixed 2 broken links

### PR DESCRIPTION
**Pull request by lwandz13**
Closes https://github.com/grafana/technical-documentation/issues/546

Fixed 2 links, one under alerting and the other regarding cross-account observability

Changed some wording for readability and clarity

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,28 +19,28 @@ build dashboards or use Explore with X-Ray to look at traces, analytics, or insi
...
 ### IAM Roles
 
-Currently all access to X-Ray is done server side by the Grafana backend using the official AWS SDK. If your Grafana
+Currently all access to X-Ray is done server-side by the Grafana backend using the official AWS SDK. If your Grafana
 server is running on AWS you can use IAM Roles and authentication will be handled automatically.
 
 See the AWS documentation on [IAM Roles](http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/iam-roles-for-amazon-ec2.html)
 
-> **Note:** [AWS Role Switching](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_use_switch-role-cli.html) is not supported at the moment.
+> **Note:** [AWS Role Switching](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_use_switch-role-cli.html) is not currently supported.
```
Comment (chri2547) [inline README.md:35]: @lwandz13 , I'm generally in favor of not including adverbs in tech docs as they are not always necessary. Up to you, just a nit.

File: README.md [hunk 2]
```diff
@@ -186,13 +186,13 @@ Since X-Ray queries can return numeric data, alerts are supported. See the [Aler
 
 ## Configure the data source with provisioning
 
-You can configure data sources using config files with Grafana's provisioning system. You can read more about how it works and all the settings you can set for data sources on the [provisioning docs page](https://grafana.com/docs/grafana/latest/administration/provisioning/).
+You can configure data sources using config files with Grafana's provisioning system. For ,ore information regarding how it works and all available settings, see [Provision Grafana](https://grafana.com/docs/grafana/latest/administration/provisioning/).
```
Comment (chri2547) [inline README.md:189]:
```suggestion
You can configure data sources using config files with Grafana's provisioning system. For ,ore information regarding how it works and all available settings, see [Provision Grafana](/docs/grafana/latest/administration/provisioning/).
```

File: README.md [hunk 3]
```diff
@@ -156,7 +156,7 @@ To display the service map:
 - Use [Explore](https://grafana.com/docs/grafana/latest/explore/) in Grafana 7.4 plus.
 - Show the data in a simple table. This is the only option if the Node graph panel is unavailable.
 
-You can pan and zoom the view with buttons or you mouse. For details about the visualization, refer to [Node graph panel](https://grafana.com/docs/grafana/latest/panels/visualizations/node-graph/).
+You can pan and zoom the view with buttons or use your mouse. For details about the visualization, see [Node graph panel](https://grafana.com/docs/grafana/latest/panels/visualizations/node-graph/).
```
Comment (chri2547) [inline README.md:159]:
```suggestion
You can pan and zoom the view with buttons or use your mouse. For details about the visualization, see [Node graph panel](/docs/grafana/latest/panels/visualizations/node-graph/).
```

File: README.md [hunk 4]
```diff
@@ -156,7 +156,7 @@ To display the service map:
 - Use [Explore](https://grafana.com/docs/grafana/latest/explore/) in Grafana 7.4 plus.
```
Comment (chri2547) [inline README.md:156]:
```suggestion
- Use [Explore](/docs/grafana/latest/explore/) in Grafana 7.4 plus.
```

**Review by chri2547 — COMMENTED** [review]
A couple of suggestions.

**Review by chri2547 — APPROVED** [review]

**Review by fridgepoet — APPROVED** [review]

**Pull request merged**